lewis/examples/simple_device/__init__.py

```diff
@@ -67,3 +67,6 @@ def set_param(self, new_param):
 
     def handle_error(self, request, error):
         return 'An error occurred: ' + repr(error)
+
+
+framework_version = '>=1.0.3'
```

Comment:
This should probably be reduced to `1.0.2` for now, like the others, and the ranging removed.